src/x-terminal.js

```diff
@@ -253,6 +253,7 @@ class XTerminalSingleton {
 			if (cursor) {
 				const line = editor.lineTextForBufferRow(cursor.row)
 				selectedText = line
+				editor.moveDown(1)
```

Comment:
I see youve restored the cursor auto move down, while for me its good, I still think the better design is to make that behavior optionally available.

run selected text is ok for moving down, but just insert text could possibly be not desired or could be desired depending on scenario.

Reply:
Ya I figured we could pickup that discussion after figuring every thing else out. For now we can assume it will work the same way as other terminals.

Reply:
its here https://github.com/bus-stop/x-terminal/pull/23#discussion_r392173865 and https://github.com/bus-stop/x-terminal/pull/23#discussion_r392182501